app/service/rest.py

```diff
@@ -297,7 +297,7 @@ def update_service(service_id):
         if not fetched_service.restricted:
             _warn_service_users_about_message_limit_changed(service_id, current_data)
     if sms_limit_changed:
-        redis_store.delete(sms_daily_count_cache_key(service_id))
+        delete_daily_sms_fragment_count(service_id)
```

Comment:
Why does this get deleted when the limit changes?  Doesn't this number represent the number of fragments sent today?

Reply:
yeah.... I had copied the code around the general daily limit, but that cache key is called "daily_limit_cache_key" and it looks like Antoine thought it was the cache key for the daily limit (silly Antoine) and so he added code here to delete it https://github.com/cds-snc/notification-api/pull/1258

But that's just a mistake caused by our bad naming